packages/cli/src/Generate.ts

```diff
@@ -138,7 +138,7 @@ ${chalk.bold('Examples')}
         printDownloadProgress: !watchMode,
         version: enginesVersion,
         cliVersion: pkg.version,
-        dataProxy: !!args['--data-proxy'],
+        dataProxy: !!args['--data-proxy'] || !!process.env.PRISMA_CLI_GENERATE_DATAPROXY,
```

Comment:
Are we testing `--data-proxy` here in any way? Can we add one for `PRISMA_CLI_GENERATE_DATAPROXY` then as well? (Fine if not, then we need to via ecosystem-tests)

Reply:
Outstanding task for post GA, only tested via ecotests right now.

Reply:
(...but `--data-proxy` is tested via `getGenerators`)

Reply:
So in theory we could replicate that test with `--data-proxy` but with an env var?